lightning/pkg/server/lightning.go

```diff
@@ -356,7 +356,8 @@ func (l *Lightning) RunOnceWithOptions(taskCtx context.Context, taskCfg *config.
 		return err
 	}
 
-	taskCfg.TaskID = time.Now().UnixNano()
+	taskCfg.TaskID = int64(uuid.New().ID())
```

Comment:
I see `uuid.New` just read some random bytes from `rand.Reader`, and `.ID()` only uses the first 4 bytes. We can directly use https://pkg.go.dev/math/rand#Int63 I think.